### PR TITLE
fix: coingecko on main manifest

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts [hunk 1]
```diff
@@ -27,14 +29,25 @@ export class CoingeckoFetcher extends BaseFetcher {
...
     const { coingeckoApiUrl, coingeckoApiKey } = config;
-    const response = await axios.get<SimplePrices>(coingeckoApiUrl, {
-      params: {
-        ids: ids.join(","),
-        vs_currencies: "usd",
-        ...(coingeckoApiKey && { x_cg_pro_api_key: coingeckoApiKey }),
-      },
-    });
-    return response.data;
+    const fetchesRequiredCount = Math.ceil(ids.length / COINGECKO_TOKENS_LIMIT);
```
Comment (hatskier) [inline packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts:32]: What do you think about using _.chunk method here to simplify the code?

File: packages/oracle-node/manifests/single-source/coingecko.json [hunk 2]
```diff
@@ -10,212 +10,458 @@
   },
   "defaultSource": ["coingecko"],
   "tokens": {
-    "AR": {},
```
Comment (hatskier) [inline packages/oracle-node/manifests/single-source/coingecko.json:13]: Why did you change the coingecko manifest?

File: packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts [hunk 3]
```diff
@@ -27,14 +29,21 @@ export class CoingeckoFetcher extends BaseFetcher {
...
-    return response.data;
+    const idsChunks = _.chunk(ids, COINGECKO_TOKENS_LIMIT);
+
+    let mergedPrices: SimplePrices = {};
+    for (const idsChunk of idsChunks) {
+      const response = await axios.get<SimplePrices>(coingeckoApiUrl, {
+        params: {
+          ids: idsChunk.join(","),
+          vs_currencies: "usd",
+          ...(coingeckoApiKey && { x_cg_pro_api_key: coingeckoApiKey }),
```
Comment (hatskier) [inline packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts:40]: What do you think about using ?? instead of && here?

**Review by hatskier — APPROVED** [review]

File: packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts (hunk 3 above)
Comment (cehali) [reply packages/oracle-node/src/fetchers/coingecko/CoingeckoFetcher.ts:40]: image
